sql/01-create-table.sql

```diff
@@ -14,7 +14,7 @@ CREATE TABLE allocations(
     size BIGINT NOT NULL DEFAULT 0,
     used_size BIGINT NOT NULL DEFAULT 0,
     owner_id VARCHAR(64) NOT NULL,
-    owner_public_key VARCHAR(256) NOT NULL,
+    owner_public_key VARCHAR(512) NOT NULL,
```

Comment:
This change should probably be an alter statement in a new file. I think we need to document how we handle database changes, currently on the info i am operating on there doesnt seem to be a way to protect against breaking changes.

Reply:
is that in a new sql file? Is this something you could submit since I'm not clear on how the sql/ directory here should be structured.

whatever gets the varchar limit to be 512 in the end, I'll be happy with it.

Reply:
You will have to add a new sql file just like all the others

Reply:
OK please check the new files now.